README.md

```diff
@@ -234,6 +234,8 @@ You can configure your dashboard for Basic Authentication by adding usernames an
 }
 ```
 
+You can store the password in either `plain text` or `bcrypt` formats. You can encrypt the password using any online bcrypt tool.
+
```

Comment:
That one looks more legit, but, I was using [this](https://www.dailycred.com/article/bcrypt-calculator) one to test as you can select the number of rounds. Thoughts?